celery/canvas.py

```diff
@@ -468,10 +463,9 @@ def __json__(self):
     def __repr__(self):
         return self.reprcall()
 
-    if JSON_NEEDS_UNICODE_KEYS:  # pragma: no cover
-        def items(self):
-            for k, v in dict.items(self):
-                yield k.decode() if isinstance(k, bytes) else k, v
+    def items(self):
+        for k, v in dict.items(self):
+            yield k.decode() if isinstance(k, bytes) else k, v
```

Comment:
looks like here is missing yield the `v` if is instance of bytes.

Reply:
That may be the case but we need to fix this on master if it is so.